Makefile

```diff
@@ -142,6 +142,13 @@ generate: deps
 
 msgp: $(patsubst %,%/msgp_gen.go,$(MSGP_GENERATE))
 
+api:
```

Comment:
Should these be run as part of the default make invocation as well?

Reply:
I don't know, it's a reasonable suggestion. I often forget to do so. But at least now it's convenient, I don't have to recall the exact directory and duck in there for a quick `make`.